examples/hosts/app-integration/schema-upgrade/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@fluid-example/app-integration-schema-upgrade",
+  "version": "0.58.2000",
+  "description": "Using external data to initialize the container state and serialize out afterwards.",
+  "homepage": "https://fluidframework.com",
+  "repository": "https://github.com/microsoft/FluidFramework",
```

Comment:
This probably needs to be updated to align with https://github.com/microsoft/FluidFramework/pull/9410